weaviate/collection/classes/internal.py

```diff
@@ -27,7 +33,46 @@ class _Object(Generic[P]):
     metadata: _MetadataReturn
 
 
-Reference: TypeAlias = List[_Object[Properties]]
+class Reference(Generic[P]):
+    def __init__(
+        self,
+        objects: Optional[List[_Object[P]]],
+        target_collection: Optional[str],
+        uuids: Optional[UUIDS],
+    ):
+        self.objects = objects
+        self.__target_collection = target_collection if target_collection else ""
+        self.__uuids = uuids
```

Comment:
hmm I think this is confusing and people will use the __init__ directly - can we solve this differently?

Reply:
hmmmm ReferenceTo also does not really sove it, as the type hint would still be `Reference` and the same could happen as now.

Then let's leave it for now and see if users are getting this right or not